waku/node/v2/waku_types.nim

```diff
@@ -154,3 +174,12 @@ proc generateRequestId*(rng: ref BrHmacDrbgContext): string =
   var bytes: array[10, byte]
   brHmacDrbgGenerate(rng[], bytes)
   toHex(bytes)
+
+proc computeIndex* (w: WakuMessage): Index=
+  ## Takes a WakuMessage and returns its index
+  let payloadStr=w.payload.join("") #convers payload to string
+  var data: string
+  data.add(w.contentTopic)
+  data.add(payloadStr)
+  result.digest= $secureHash(data) #computes sha1 of data
+  result.timeStamp= $getTime().utc #gets the timestamp in utc with seconds precision
```

Comment:
What about `epochTime`, wouldn't that be easier to work with?

Reply:
I can also use `epochTime`, it gives a `float` value, however, the documentation of `epochTime` proc recommends  `getTime` instead of `epochTime`:
> "``getTime`` should generally be preferred over this proc."
I am not sure why.

But storage-wise, float is better than string.